src/server/routes/core.ts

```diff
@@ -16,6 +16,14 @@ router.use(
   }),
 );
 
+router.use(
+  '/.well-known',
+  express.static(path.resolve(__dirname, '.well-known'), {
+    cacheControl: true,
+    maxAge: ms('1y'),
```

Comment:
hmm, what if they needed to make changes to this file ? It's probably worth setting this to 30 mins so we don't need to flush the cache in fastly if there is a prod change

Reply:
updated it to 30m and left a comment about this